test/benchmark/utils/parser.py

```diff
@@ -649,12 +652,23 @@ def write_compare_list_to_js(log_dir_str, case_information, output_filename):
         f.write(json_str)
     pass
 
+def copy_html_to_output_dir(output_dir):
+    for folder in ['report', 'static']:
+        path_src = os.path.join(path_html, folder)
+        path_dest = os.path.join(output_dir, folder)
+        try:
+            shutil.copytree(path_src, path_dest)
+        except Exception:
```

Comment:
Do we know exactly what exception could happen ? or we just want to catch everything?